schema/transportation/segment.yaml

```diff
@@ -76,15 +76,13 @@ properties:
                 If a lane is restricted for use only by HOVs, then the default minimum vehicle
                 occupancy qualifying as HOV is 2+ passengers (including the driver).  This
                 restriction can be provided to override the default definition of HOV.
-              oneOf:
-                - "$ref": "#/$defs/propertyDefinitions/relationalExpression"
-                - type: array
-                  items:
-                    type: object
-                    unevaluatedProperties: false
-                    allOf:
-                      - "$ref": "#/$defs/propertyContainers/ruleContainer"
-                      - "$ref": "#/$defs/propertyDefinitions/relationalExpression"
+              type: array
+              items:
+                type: object
+                unevaluatedProperties: false
+                allOf:
+                  - "$ref": "#/$defs/propertyContainers/ruleContainer"
+                  - "$ref": "#/$defs/propertyDefinitions/relationalExpression"
```

Comment:
I think this one is more badly broken than I previously thought. (My apologies @TheRBajaj, this isn't your issue, this is just a conceptual problem with the schema, but we should fix it.)

The whole `relationalExpression` thing makes no sense here.

*i.e.* It makes no sense to have rules like:

```yaml
min_occupancy:
 - is_at_least: 3
```

That's because a `min` or minimum already implies that something "is at least". Also, if we allow relational, we can get absurdity:

```yaml
min_occupancy:
 - is_at_most: 3
```

That doesn't make any sense: it's putting a maximum limit on a minimum!

I think what we need, and again apologies @TheRBajaj I didn't know this aspect of the schema was in such rough shape, is to dump the whole `relationalExpression` thing and just have a `value` property, which must be an integer, of value at least `1`.

So the rules should look like:

```yaml
min_occupancy:
 - value: 3
 - when: { mode: [truck] }
   value: 4
```

Could you make this change? It basically requires:

1. [ ] Drop current line 85.
2. [ ] Add a `properties` section at line `85` that has one field, `value`, that has integer type and a `minimum` of `1`.
3. [ ] Repair the examples/counter-examples accordingly.

Reply:
got it, not a problem at all! will make the necessary changes.